server/controllers/temporary-accommodation/manage/assessmentsController.ts

```diff
@@ -40,6 +40,33 @@ export const archivedAssessmentsTableHeaders = [
     },
   },
 ]
+export const confirmationPageContent: Record<string, Record<'title' | 'text', string>> = {
+  in_review: {
+    title: 'Mark this referral as in review',
+    text: '<p class="govuk-body">Mark this referral as in review if you or a HPT colleague are working on this referral.</p>',
+  },
+  ready_to_place: {
+    title: 'Confirm this referral is ready to place',
+    text: `<p class="govuk-body">Mark this referral as ready to place to find a suitable bedspace.</p>
+      <p class="govuk-body">Some regions will require approval before a bedspace is booked. Other regions require an address before approvals can be given. Check with your manager if you are unsure.</p>
+      <p class="govuk-body">Once a person has been placed the referral should be archived.</p>`,
+  },
+  rejection: {
```

Comment:
Should this be "rejected" rather than "rejection", to match the statuses on the type?